docs/DeveloperGuide.md

```diff
@@ -1894,3 +1894,41 @@ testers are expected to do more _exploratory_ testing.
     1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
 
 1. _{ more test cases …​ }_
+
+---
+
+## **Appendix: Effort**
+
+<br>
+
+### Difficulty level
+
+This project was moderately challenging as we had to deal with a large existing code base.
+It took us some time to understand how the different components interact with each other.
+Furthermore, our team was not familiar with frameworks such as JavaFX prior to this, and we had to learn how to use it.
+
+<br>
+
+### Challenges faced
```

Comment:
Sounds good, but perhaps we could add the challenge of figuring out how to hide the app behind a login / logout system where the password is hashed and how this account data has to be stored? Something like that haha

Reply:
Added!